app/assets/javascripts/application.js

```diff
@@ -21,4 +21,71 @@
 //= require gmaps/google
 //= require_tree .
 
-$(function(){ $(document).foundation('tab', 'reflow'); });
+$(function(){
+  $(document).foundation('tab', 'reflow');
+
+  // checkin field set on new page
+  $('.checkin-contact select, .checkin-contact input').attr("disabled", "true");
+  $('.checkin-contact').addClass("disable-field-set");
+
+  $('.checkin-contact').click(function(){
+    $('.checkin-local').addClass("disable-field-set");
+    $('.checkin-contact').removeClass("disable-field-set");
+    $('.checkin-local input').attr("disabled", "true");
+    $('.checkin-contact select, .checkin-contact input').removeAttr("disabled");
+  });
+
+  $('.checkin-local').click(function(){
+    $('.checkin-contact').addClass("disable-field-set");
+    $('.checkin-local').removeClass("disable-field-set");
+    $('.checkin-local input').removeAttr("disabled")
+    $('.checkin-contact select, .checkin-contact input').attr("disabled", "true");
+  });
+
+
+  // group show page side bar
+  $("#side-user li").hover(function(){
+    /* This is the same for all */
+    $(".pop-up").removeClass("hide");
+    var popUpOffSet = $(this).offset();
+    popUpOffSet.left += $(this).width();
+    $(".pop-up").offset(popUpOffSet);
+    var popUpData = JSON.parse($(this).attr("meta-data"));
+    /* This is the same for all */
+
+    /* This is the different base on json object for all */
+    var wrapperDiv = document.createElement("div");
+    var nameDiv = document.createElement("div");
+    nameDiv.innerHTML = popUpData.first_name + " " + popUpData.last_name;
```

Comment:
Identifier 'first_name' is not in camel case.<br>Mixed double and single quotes.<br>Identifier 'last_name' is not in camel case.